inst/@sym/horzcat.m

```diff
@@ -62,7 +62,7 @@
           'return sp.Matrix.hstack(*_proc),'
           };
 
-  varargin = sym(varargin);
+  varargin = cell_array_to_sym(varargin);
```

Comment:
Yes I got that error message too. I just added the private dir to path... I had faced similar issues in past but could never figure out the cause. So i thought it was not related to classdef.
And, after classdef, did u try 'sym({1})'?? For me octave gave the error' cannot convert cell to object'